src/pybind/mgr/dashboard/constraints.txt

```diff
@@ -3,5 +3,5 @@ more-itertools==4.1.0
 PyJWT==2.0.1
 bcrypt==3.1.4
 python3-saml==1.4.1
-requests==2.25.1
```

Comment:
might want to introduce this change to master first, and then backport it to this branch.

Reply:
And, actually, wait an hour or so because you might be able to cherry-pick from https://github.com/ceph/ceph/pull/45647

Reply:
Will do, removed that commit and marked this PR with DNM.